macs.php

```diff
@@ -0,0 +1 @@
+i<?php include 'data.php';  ?>
```

Comment:
Hehe, ok :-) wow you still tracking #0x20? Been a long time for me...

See you on 32c3?

Greets,
Sandb

On Thu, Dec 3, 2015, 19:30 Jeroen De Dauw <notifications@github.com> wrote:

> In macs.php <https://github.com/sandb/pamela/pull/3#discussion_r46590754>:
>
> > @@ -0,0 +1 @@
> > +i<?php include 'data.php';  ?>
>
> Nope, but whoever carroarmato0 is on the 0x20 irc is :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sandb/pamela/pull/3/files#r46590754>.
>